src/main/com/cqfn/degitx/simulator/GitRepo.kt

```diff
@@ -0,0 +1,33 @@
+package com.cqfn.degitx.simulator
+
+/**
+ * DRAFT: GitRepository replica, placed at some Node
+ */
+interface GitRepo {
```

Comment:
@IlyaMoskva and same here - simulator doesn't care about format of node's state, it can be git repository or just binary files, it doesn't matter.

Reply:
@g4s8 , right, it is not needed. It was old design.